kuma/redirects/redirects.py

```diff
@@ -559,4 +559,158 @@
+        re_flags='i', prepend_locale=False, permanent=True),
+
+    # Old landing pages. The regex, adapted from Bedrock, captures locale prefixes.
+    # RewriteRule ^(\w{2,3}(?:-\w{2})?/)?mobile/?$ /$1docs/Mozilla/Mobile
+    # [R=301,L]
+    redirect(
+        r'^(?P<localeprefix>\w{2,3}(?:-\w{2})?/)?mobile/?$',
+        '/{localeprefix}docs/Mozilla/Mobile',
+        prepend_locale=True, permanent=True),
+
+    # RewriteRule ^(\w{2,3}(?:-\w{2})?/)?addons/?$ /$1Add-ons [R=301,L]
+    redirect(
+        r'^(?P<localeprefix>\w{2,3}(?:-\w{2})?/)?addons/?$',
+        '/{localeprefix}Add-ons',
+        prepend_locale=True, permanent=True),
+
+    # RewriteRule ^(\w{2,3}(?:-\w{2})?/)?mozilla/?$ /$1docs/Mozilla [R=301,L]
+    redirect(
+        r'^(?P<localeprefix>\w{2,3}(?:-\w{2})?/)?mozilla/?$',
+        '/{localeprefix}docs/Mozilla',
+        prepend_locale=True, permanent=True),
+
+    # RewriteRule ^(\w{2,3}(?:-\w{2})?/)?web/?$ /$1docs/Web [R=301,L]
+    redirect(
+        r'^(?P<localeprefix>\w{2,3}(?:-\w{2})?/)?web/?$',
+        '/{localeprefix}docs/Web',
+        prepend_locale=True, permanent=True),
+
+    # RewriteRule ^(\w{2,3}(?:-\w{2})?/)?learn/html5/?$
+    # /$1docs/Web/Guide/HTML/HTML5 [R=301,L]
+    redirect(
+        r'^(?P<localeprefix>\w{2,3}(?:-\w{2})?/)?learn/html5/?$',
+        '/{localeprefix}docs/Web/Guide/HTML/HTML5',
+        prepend_locale=True, permanent=True),
+
+    # Some blanket section moves / renames
+    # RewriteRule ^En/JavaScript/Reference/Objects/Array$
+    # en-US/docs/JavaScript/Reference/Global_Objects/Array [R=301,L,NC]
+    redirect(
+        r'^En/JavaScript/Reference/Objects/Array$',
+        '/en-US/docs/JavaScript/Reference/Global_Objects/Array',
+        re_flags='i', prepend_locale=True, permanent=True),
+
+    # RewriteRule ^En/JavaScript/Reference/Objects$
+    # en-US/docs/JavaScript/Reference/Global_Objects/Object [R=301,L,NC]
+    redirect(
+        r'^En/JavaScript/Reference/Objects$',
+        '/en-US/docs/JavaScript/Reference/Global_Objects/Object',
+        re_flags='i', prepend_locale=True, permanent=True),
+
+    # RewriteRule ^En/Core_JavaScript_1\.5_Reference/Objects/(.*)
+    # en-US/docs/JavaScript/Reference/Global_Objects/$1 [R=301,L,NC]
+    redirect(
+        r'^En/Core_JavaScript_1\.5_Reference/Objects/(?P<suffix>.*)$',
+        '/en-US/docs/JavaScript/Reference/Global_Objects/{suffix}',
+        re_flags='i', prepend_locale=True, permanent=True),
+
+    # RewriteRule ^En/Core_JavaScript_1\.5_Reference/(.*)
+    # en-US/docs/JavaScript/Reference/$1 [R=301,L,NC]
+    redirect(
+        r'^En/Core_JavaScript_1\.5_Reference/(?P<suffix>.*)$',
+        '/en-US/docs/JavaScript/Reference/{suffix}',
+        re_flags='i', prepend_locale=True, permanent=True),
+
+    # RewriteRule ^([\w\-]*)/HTML5$ $1/docs/HTML/HTML5 [R=301,L,NC]
+    redirect(
+        r'^(?P<pre>[\w\-]*)/HTML5$',
+        '/{pre}/docs/HTML/HTML5',
+        re_flags='i', prepend_locale=False, permanent=True),
```

Comment:
This pattern is causing the Kuma test failure. I'll need to look closer at Kuma and commit history to decide which needs to bend.